### ISSUE TITLE
ReferenceError: Unknown plugin "transform-vue-jsx" specified in babelrc

### ISSUE THREAD

**Issue by alvarotrigo** [alvarotrigo/vue-fullpage.js#69]
1. clone master/dev
2. open demos/webpack
3. npm i
4. npm run dev

---

result

```bash

Module build failed: ReferenceError: Unknown plugin "transform-vue-jsx" specified in "E:\\git\\clone\\vue-fullpage.js\\.babelrc" at 0, attempted to resolve relative to 

```

_Originally posted by @xiaotiandada in https://github.com/alvarotrigo/vue-fullpage.js/issues/60#issuecomment-435913756_

**Comment by alvarotrigo** [comment]
Fixed on the dev branch thanks to @Chiaparini in his pull request https://github.com/alvarotrigo/vue-fullpage.js/pull/72

**Comment by alvarotrigo** [comment]
Fixed in version 0.0.9
